pkgs/applications/misc/mdp/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchgit, ncurses }:
+
+stdenv.mkDerivation {
+  name = "mdp-0.93";
+
+  src = fetchgit {
+    url = "git://github.com/visit1985/mdp.git";
+    rev = "09d6bd1a8a33fac75a999f0822ec10cb77fbc072";
+    sha256 = "1jx9i5dzz3invs76f72jzv87cm87y2cq747pkg2f3vsni8d21cam";
+  };
+
+  installPhase = "make install PREFIX=$out";
```

Comment:
Not a style expert, but `makeFlags = "PREFIX=$(out)";` seems to be the idiomatic way to do this. Or does that mess up anotherPhase?